RELEASE.md

```diff
@@ -1,13 +1,36 @@
 # Release Process
 
-TODO
+## Overview
 
-(text below is from the template)
+The Service APIs project is an API project that has the following two components:
+- Kubernetes Custom Resource Definitions (CRDs)
+- Corresponding Go API in the form of `sigs.k8s.io/service-apis` Go package
 
-The Kubernetes Template Project is released on an as-needed basis. The process is as follows:
+This repository is the home for both of the above components.
 
-1. An issue is proposing a new release with a changelog since the last release
-1. All [OWNERS](OWNERS) must LGTM this release
-1. An OWNER runs `git tag -s $VERSION` and inserts the changelog and pushes the tag with `git push $VERSION`
-1. The release issue is closed
-1. An announcement email is sent to `kubernetes-dev@googlegroups.com` with the subject `[ANNOUNCE] kubernetes-template-project $VERSION is released`
+## Versioning strategy
+
+Releases in this repository follow Go module versioning conventions and
+semantic versioning.
+During the `alpha` and `beta` stage of the API, version tags will take the form
+of `v0.x.y`.
+Minor version must be incremented whenever a new API version is introduced for
+any resource or whenever there are substantial changes in the API.
+Bug fixes and clarifications in the spec will lead to patch number being incremented.
+
+During GA (when `apiVersion` changes to `v1`), the version tag might be bumped
+up to `v1.0.0`. This will be revised based on experience and feedback from the
+community.
+
+> The first release candidate was tagged as `v1alpha1-rc1`. It predates this
+document and is an exception.
+
+## Releasing a new version
+
+- Write the [changelog](CHANGELOG.md) with user-visible API changes. This must
+  go through the regular PR review process and get merged into the `master` branch.
```

Comment:
Maybe note that approval of this PR indicates community support for a GitHub release to be published immediately after the PR is merged.